src/ServiceContainer/SymfonyExtension.php

```diff
@@ -212,6 +212,11 @@ private function declareSymfonyContainers(ContainerBuilder $container): void
                 new KernelBasedContainerAccessor($container->get(self::KERNEL_ID))
             );
 
+            $this->crossContainerProcessor->addContainerAccessor(
+                'symfony_driver',
+                new KernelBasedContainerAccessor($container->get(self::KERNEL_ID))
```

Comment:
Shouldn't it be `self:: DRIVER_KERNEL_ID`?

Reply:
should be